ffi/diplomat/ffi_coverage/src/main.rs

```diff
@@ -204,6 +206,22 @@ lazy_static::lazy_static! {
         "icu::properties::SentenceBreak",
         "icu::properties::WordBreak",
 
+        // Experimental
+        "icu::properties::maps::load_canonical_combining_class",
+
+        // Unnecessary for now
+        "icu::properties::maps::CodePointMapDataBorrowed::iter_ranges",
+        "icu::properties::sets::CodePointSetDataBorrowed::iter_ranges",
+
+        // Not planned until someone needs them
```

Comment:
Issue: I know we said we would punt on these, but I do feel that these APIs absolutely belong in FFI. They are part of the ECMA-402 specification. Please file a follow-up issue for 1.x rather than leaving it as an unactioned comment.

Reply:
This _entire_ section of ignores is for things that fall under that category. "Not for 1.0 but to be done later" is _exactly_ what it's for, and in 1.x we shall need to revisit the whole section and plan it. Just because it's in an ignorelist doesn't mean it's gone forever; the point of this section of the ignorelist is to make life easy for 1.0 and we look at it again.

I do not think it would  be productive to file an issue for these cases right now, we probably should file an issue covering the _entire_ section. and make sure we go back and plan individual pieces post-1.0

Reply:
https://github.com/unicode-org/icu4x/issues/2492